<launch>
  <include file="$(find autorally_core)/launch/hardware.machine" />

  <node name="xbeeNode" pkg="autorally_core" type="xbeeNode"
        machine="autorally-master" output="screen">

    <param name="frameID" type="string" value="41"/>
    <param name="diagnosticInfo" value="DB %V ER GD TR TP AP AO ID NI SH SL CE" />
    <param name="transmitPositionRate" value="10"/>

    <param name="port" value="/dev/arXbeeNode" />
    <!--configure settings for 230400 baud, 8N1 -->
    <param name="serialBaud" value="230400" />
    <param name="serialDataBits" value="8" />
    <param name="serialParity" value="none" />
    <param name="serialStopBits" value="1" />
    <param name="serialHardwareFlow" value="true" />
    <param name="serialSoftwareFlow" value="false" />
  </node>
</launch>
